.gitignore

```diff
@@ -69,6 +69,9 @@ ipch
 # sphinx generated files
 /docs/build/
 
+# cmake generated files
+build/
```

Comment:
This subsumes the `/build/gyp` rule on line 45. You can remove that one. Can you also add a leading slash?

Reply:
ok